obal/data/roles/build_package/tasks/waitrepo.yml

```diff
@@ -0,0 +1,14 @@
+- block:
+  - name: "Get {{ build_package_koji_command }} build targets"
+    command: "{{ build_package_koji_command }} taskinfo -v {{ item }}"
+    with_items: "{{ build_package_koji_tasks }}"
+    register: build_package_koji_task_info
+
+  - name: "Filter {{ build_package_build_system }} output"
+    set_fact:
+      build_package_koji_task_data: "{{ build_package_koji_task_info.results | map(attribute='stdout') | map('from_yaml') | list }}"
+
+  - name: "Wait for {{ build_package_koji_command }} to regenerate the repo"
+    command: "{{ build_package_koji_command }} wait-repo --build={{ item['Build'] | regex_replace(' .*', '') }} {{ item['Request Parameters']['Build Target'] | regex_replace('-candidate$', '') }}-build"
+    with_items: "{{ build_package_koji_task_data }}"
+  when: build_package_build_system == 'koji' and build_package_waitrepo|bool
```

Comment:
Can this be placed above the block? e.g.

```
- when: blahl blahb blahb
  block:
    -
```

I feel like given yaml this should work and is in my opinion much more readable than the dangling when

Reply:
My memory says it does not, but I'll try out to be sure tomorrow.

Reply:
It did \o/